### ISSUE TITLE
ion-pullup-tab swipper is not working

### ISSUE THREAD

**Issue by lianesten** [arielfaur/ionic-pullup#37]
Great job!
I implemented ionic-pullup into my ionic 2 v2.2.1 application, but the ion-pullup-tab swipper is not working. I tested in android physical device, emulators and browser. Any idea why is not working?
Thanks!!

**Comment by arielfaur** [comment]
what do you mean with swipper?

**Comment by lianesten** [comment]
@arielfaur I mean the up-down vertical gesture touching the footer tab.
Something similar to the image:
![image](https://cloud.githubusercontent.com/assets/6086119/25102309/98e5b998-237d-11e7-9218-fb2feaaf4dc4.png)

**Comment by arielfaur** [comment]
Yes, it was disabled on purpose because it causes some issues. I am looking into that. In the meantime only the tap gesture works.

**Comment by jorgemejia** [comment]
Good component, great job!!

Is it still disabled ? It would be great if this gesture works.

**Comment by SaltyPotato** [comment]
Hello, I'm also interested in this feature. Is it still disabled?
